portfolio/src/main/webapp/index.html

```diff
@@ -8,15 +8,27 @@
   </head>
   <body>
     <div id="content">
-      <h1>Steven's Portfolio</h1>
-      <img src="Images/MeBAW.jpeg" width="431" height="306"/>
-      <p>This is my portfolio.</p>
-      <button onclick="getGreetings()">Click Me</button>
-      <div id="hello-container"></div>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Steven's Portfolio</h1>
+        <img src="Images/MeBAW.jpeg" width="431" height="306"/>
+        <p>This is my portfolio.</p>
+        <button onclick="getGreeting()">Click Me</button>
+        <div id="hello-container"></div>
+        <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
+
+        <form action="/data" method="POST">
+
+            <p>Enter your comment here:</p>
+            <textarea name="text-input">Your text here</textarea>
+            <br/><br/>
+
+            <input type="submit" />
+        </form>
+        <button onclick="getComments()">Click Me</button>
+        <div id="comments-container"></div>
     </div>
```

Comment:
It looks like the indentation got mixed up somewhere in here. There shouldn't be 4 spaces between groupings.